server/jobs/helpers.js

```diff
@@ -16,7 +16,10 @@ export const getMailInfo = async (placement) => {
     end_date: rollOffDate,
     start_date: dateStart,
   } = placement;
-  const { location: partnerLocation } = await findPartnerById(partnerId);
+
+  // const { location: partnerLocation } = await findPartnerById(partnerId);
```

Comment:
Is this not supposed to be fixed in this PR?

Reply:
It was previously another story but am looking to solve this in this PR